tests/atoms/gaia_data_layer.js

```diff
@@ -43,46 +43,49 @@ var GaiaDataLayer = {
     marionetteScriptFinished(true);
   },
 
-  disableBluetooth: function() {
+  setBluetooth: function(aValue) {
```

Comment:
I'll use  `aState` then.  `expectedState` is more for comparing values, while here I'm using an argument.